.jscsrc

```diff
@@ -129,7 +129,7 @@
     //     "beforeOpeningCurlyBrace": true
     // },
     // "requireSpacesInsideArrayBrackets": false,
-    "requireSpacesInsideObjectBrackets": "allButNested",
+    // "requireSpacesInsideObjectBrackets": "allButNested",
```

Comment:
With the existing rules, neither ``{xyz: value}`` nor ``{ xyz: value }`` is valid. Commenting this rule allows the former to work. I wanted to avoid having new lines in the middle of function arguments.